### PR TITLE
nc is closed by the defer statement.

### PR DESCRIPTION
**Pull request by tmcnicol**

### CODE REVIEW

**Comment by gcolliso** [comment]
@kozlovic could you review just to be sure this is accurate? Thank you!

File: developing-with-nats/events/README.md [hunk 1]
```diff
@@ -25,8 +25,6 @@ getStatusTxt := func(nc *nats.Conn) string {
 }
 log.Printf("The connection is %v\n", getStatusTxt(nc))
 
-nc.Close()
```
Comment (kozlovic) [inline developing-with-nats/events/README.md:28]: The connection is closed by defer statement, but we explicitly close here to show the difference of the status. See how we print the status before and after the close to show the difference.

**Comment by kozlovic** [comment]
It is true that there is a defer, but we want an explicit close here to show the status before and after the close.

File: developing-with-nats/events/README.md (hunk 1 above)
Comment (tmcnicol) [reply developing-with-nats/events/README.md:28]: yeah that makes sense

**Pull request closed without merging**